ProcessLib/GroundwaterFlowProcess.h

```diff
@@ -146,7 +127,7 @@ class GroundwaterFlowProcess : public Process
     {
         DBUG("Create local assemblers.");
         // Populate the vector of local assemblers.
-        _local_assemblers.resize(_mesh.getNElements());
+        _local_assemblers.resize(this->_mesh.getNElements());
```

Comment:
Sure, later on most of the `this->` will vanish, but if there will be some of those constructs left, it might be advisable to rather use `Process<>::` in order to not encourage users to use `this->` all over the place.